components/wso2is.key.manager.core/src/main/java/org/wso2/is/key/manager/core/observers/ReservedUserCreationObserver.java

```diff
@@ -66,9 +66,15 @@ public void createReservedUser() {
                 }
                 boolean isReservedUserCreated = userStoreManager.isExistingUser(DEFAULT_RESERVED_USERNAME);
                 if (!isReservedUserCreated) {
-                    userStoreManager.addUser(DEFAULT_RESERVED_USERNAME, getSaltString(),
-                            new String[]{EVERYONE_ROLE},
-                            new HashMap<>(), DEFAULT_RESERVED_USERNAME, false);
+                    try {
+                        userStoreManager.addUser(DEFAULT_RESERVED_USERNAME, getSaltString(),
+                                new String[]{EVERYONE_ROLE},
+                                new HashMap<>(), DEFAULT_RESERVED_USERNAME, false);
+                    } catch (UserStoreException ex) {
+                        log.warn("Unable to create the reserved user. Please create a user by" +
```

Comment:
can we specifically catch the error code according to the user

Reply:
resolved